r/R/arrow-package.R

```diff
@@ -108,7 +108,7 @@ ArrowObject <- R6Class("ArrowObject",
 }
 
 #' @export
-all.equal.ArrowObject <- function(target, current, ..., check.attributes = TRUE) {
+all.equal.ArrowObject <- function(target, current, ..., check.attributes = FALSE) {
```

Comment:
Ah, I see, you did change this, I missed that. Yeah I'd revert this. `all.equal` isn't typically called directly by a human.